docs/development/deploying.md

```diff
@@ -0,0 +1,33 @@
+# Deploying the operator pod
+
+The following set of steps highlight how to deploy a Build operator pod into an existing Kubernetes cluster.
+
+1. Build a custom docker image from this repository. This can be done with Docker, for example:
+
+   ```sh
+   pushd $GOPATH/src/github.com/redhat-developer/build
+   docker build -t eeeoo/build-operator:master .
+   docker push eeeoo/build-operator:master
+   popd
+   ```
+   Just to illustrate the above, you can find the image in [dockerhub](https://hub.docker.com/repository/docker/eeeoo/build-operator)
+
+2. Reference the generated image name in the [operator.yaml](../../deploy/operator.yaml). The `spec.template.containers[0].image` value should be modified.
+
+3. Target your Kubernetes cluster and install the Tekton pipeline controller:
```

Comment:
I think we just need to say `install the Tekton pipeline`, because there are Tekton pipeline controller and webhook, we need to install both of these two.

Reply:
done